docs/source/scilla-in-depth.rst

```diff
@@ -294,23 +294,25 @@ multiple parameters are separated by ``,``.
 
 .. note::
 
-   Transition parameters must be of a *serialisable* type:
+    Transition parameters must be of a *serialisable* type.
 
-   - Messages, events and the special ``Unit`` type are not
-     serialisable.
+    Serialisable:
+    
+    - Byte strings are serialisable.
 
-   - Byte strings are serialisable. Addresses are serialisable only as
-     ``ByStr20`` values. All other primitive types like integers and
-     strings are serialisable.
+    - Addresses are serialisable only as ``ByStr20`` values.
 
-   - Function types and map types are not serialisable.
+    - All other primitive types like integers and strings are serialisable.
```

Comment:
This needs to be rephrased. Messages, events and the unit type are considered primitive types in Scilla, so they are captured by this statement, yet they are not serialisable.

Reply:
`Byte strings, Integers, strings, block numbers are serialisable.`
How about this?

Reply:
Should be `and block numbers`, but otherwise fine.

Reply:
sure